btcjson/btcwalletextcmds.go

```diff
@@ -93,6 +93,84 @@ func NewRenameAccountCmd(oldAccount, newAccount string) *RenameAccountCmd {
 	}
 }
 
+// PsbtInput represents an input to include in the PSBT created by the
+// WalletCreateFundedPsbtCmd command.
+type PsbtInput struct {
+	Txid     string `json:"txid"`
+	Vout     int64  `json:"vout"`
+	Sequence int64  `json:"sequence"`
+}
+
+// PsbtOutput represents an output to include in the PSBT created by the
+// WalletCreateFundedPsbtCmd command.
+type PsbtOutput map[string]float64
+
+// NewPsbtOutput returns a new instance of a PSBT output to use with the
+// WalletCreateFundedPsbtCmd command.
+func NewPsbtOutput(address string, amount float64) PsbtOutput {
+	out := make(map[string]float64)
+	out[address] = amount
+	return out
+}
+
+// WalletCreateFundedPsbtOpts represents the optional options struct provided
+// with a WalletCreateFundedPsbtCmd command.
+type WalletCreateFundedPsbtOpts struct {
```

Comment:
This is exactly the same as `FundRawTransactionOpts`, so wondering if we should reuse it.

I would have recommended renaming `FundRawTransactionOpts` to something generic and using it here as well, but that would be a breaking change.

@Rjected What's your opinion on this?

Reply:
It's possible this could just be an alias of `FundRawTransactionOpts`, and then we could make changes to `FundRawTransactionOpts` if necessary, so that's one option but I don't think it's the best way to do it. We _could_ rename the struct, and that would be breaking. On the other hand, we don't really reuse btcjson option structs (afaik) and the `fundrawtransaction` / `walletcreatefundedpsbt` apis are probably not going to change too much. I'm thinking it might be okay if we leave this as is, and update the structs independently if there are any future additions to their options. Overall I don't think it's too big of a deal if we duplicate some code or have some identical structs that represent different options in btcjson, especially if the alternative is a breaking change. WDYT?

Reply:
So it's finally down to alias vs duplicating code. I'm fine with both - the former is less code to manage, and the latter has no code coupling.

If we duplicate the code, can we agree to share some common enum types (ex: a type for `ChangeType`)?

Reply:
Agree with @Rjected. Even though they are share the same structure today, I don't think these structs represent semantically the same type, and they may very well vary in the future as the inputs for either one of these methods are changed. There are also likely very few reasons to want to share the same input for calling both methods.

Sharing the type for `ChangeType` makes sense though.

Reply:
Yep, this makes sense